kolibri/core/assets/src/views/ExamReport/CurrentTryOverview.vue

```diff
@@ -53,7 +61,9 @@
         {{ $tr('attemptedLabel') }}
       </th>
       <td>
-        <ElapsedTime :date="completionTimestamp" />
+        <ElapsedTime
+          :date="new Date(currentTry.completion_timestamp || currentTry.end_timestamp)"
```

Comment:
Yes - and just in case there's any data weirdness, try to handle an edge case. end_timestamp is a required field but completion_timestamp is optional, so this should cover our basis.